installer/scripts/service_control

```diff
@@ -191,7 +202,9 @@ find_systemd_dir()
 
 enable_omsagent_service()
 {
-    if [ ! -f /etc/opt/microsoft/omsagent/conf/.service_registered ] && [ -f /etc/opt/microsoft/omsagent/conf/omsadmin.conf ]; then
+    warn_if_agent_not_onboarded
```

Comment:
Since we have a test just below, shouldn't this just be in the `else` case for the `if` at line 207?